parsedatetime/pdt_locales/en_A.py

```diff
@@ -0,0 +1,27 @@
+# -*- encoding: utf-8 -*-
```

Comment:
I got name files from ```self.localeID``` variable.
I think for ```pdtLocale_au``` this variable should be equal ```en_AU```, but compatibility....